libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/mockResultsTab/mockResultsTab.tsx

```diff
@@ -37,24 +43,83 @@ export const MockResultsTab = () => {
   const mocks = useMocksByOperation(nodeName);
   const mocksValidationErrors = useMocksValidationErrors();
 
+  const [errorMessage, setErrorMessage] = useState<string>(mocks.errorMessage || '');
```

Comment:
I feel that you don't need the states here anymore. As you are saving everything under the react redux. You just need to replace the instances of `errorMessage` to `mocks.errorMessage` and `errorCode` to `mocks.errorCode`